src/app/app-routing.module.ts

```diff
@@ -3,15 +3,20 @@ import { RouterModule, Routes } from '@angular/router';
 import { MarkingDetailsComponent } from './marking-details/marking-details.component';
 import { HomeComponent } from './home/home.component';
 import { AuthenticationComponent } from './authentication/authentication.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
+import { AuthGuard } from './auth.guard';
+
 
 const routes: Routes = [
-    { path: 'authentication', component: AuthenticationComponent },
+    { path: 'authentication', component: AuthenticationComponent},
+    {path: 'dashboard', component: DashboardComponent,canActivate:[AuthGuard]},
     { path: ':id', component: MarkingDetailsComponent },
     { path: '', component: HomeComponent }
 ];
 
 @NgModule({
     imports: [RouterModule.forRoot(routes)],
-    exports: [RouterModule]
+    exports: [RouterModule],
+    providers: [],
```

Comment:
Is there a reason for adding this? If not please remove.

Reply:
Removing it right away

Reply:
I think this is some visual bug because on my end it's all good